systemtest/src/test/java/io/strimzi/systemtest/migration/MigrationST.java

```diff
@@ -149,13 +206,90 @@ void testMigrationFromZkToKRaft() {
     @IsolatedTest
     void testRollbackDuringMigration() {
         TestStorage testStorage = new TestStorage(ResourceManager.getTestContext(), TestConstants.CO_NAMESPACE);
+        boolean zkDeleteClaim = true;
+        boolean deleteCoDuringProcess = false;
```

Comment:
can't we just pass the boolean(s) instead of declaring these local variables?

Reply:
yeah we can, just in terms of the review and clarity I left it there